packages/swap/src/event-handlers/swapDepositAddressReady.ts

```diff
@@ -79,7 +79,6 @@ export const swapDepositAddressReady = async ({
         },
       },
       create: {
-        expectedDepositAmount: 0,
```

Comment:
i think its better to make it explicit that we dont know what the user intends to send. before this change, we exposed `"expectedDepositAmount": 0` from the status in the SDK